### PR TITLE
Updated to Latest version of ember CLI 3.7.1

### PR DESCRIPTION
**Pull request by Mike-Neto**
Motivated by the amount of vulnerabilities adding this addon brought to my own app.

All these new files are core to the new ember version.
The only manual updates I made were to tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,4 +32,10 @@ var app = new EmberApp({
 });
 ```
 
+## Troubleshooting
+
+You must have phantomjs installed in your system for the tests to run.
```
Comment (andybluntish) [inline README.md]: I don't think this is true anymore, I can run them fine without it.

**Comment by andybluntish** [comment]
Thank you very much—this is long overdue 👍

File: README.md (hunk 1 above)
Comment (Mike-Neto) [reply README.md]: It is if you don't have chrome installed, which I didn't since I use chromium under Linux.

**Comment by Mike-Neto** [comment]
Can we get a version bump & Release?

**Pull request merged**